.cirrus.yml

```diff
@@ -14,12 +14,12 @@ env:
     ####
     #### Cache-image names to test with (double-quotes around names are critical)
     ####
-    FEDORA_NAME: "fedora-36"
-    PRIOR_FEDORA_NAME: "fedora-35"
+    FEDORA_NAME: "fedora-37"
+    PRIOR_FEDORA_NAME: "fedora-36"
     UBUNTU_NAME: "ubuntu-2204"
 
     # Google-cloud VM Images
-    IMAGE_SUFFIX: "c6193881921355776"
+    IMAGE_SUFFIX: "c5069932136759296"
```

Comment:
@rhatdan just curious where you got the image id from

Reply:
AFAICT the latest image built was at https://github.com/containers/automation_images/pull/241 , which has `cache	fedora-podman-py	c4987131005763584` . So I think that would be the one to try unless I'm missing something.

Reply:
from podman.

Reply:
Trying yours